src/hatti/views/table.cljs

```diff
@@ -82,21 +83,47 @@
   (let [clj-value (js->clj value :keywordize-keys true)]
     (forms/format-answer field clj-value language true)))
 
+(defn button-formatter
+  [row cell value columnDef dataContext]
+  (let [{:keys [owner project formid]} (:dataset-info @shared/app-state)
+        form-owner (last-url-param owner)
+        project-id (last-url-param project)
+        edit-link (str "/"
+                       (url form-owner project-id formid
+                            "webform?instance-id=")
+                       value)]
+    (str
```

Comment:
yes, @okal wrote a function for this in Zebra but we'll need to copy it to hatti utils to make use if it https://github.com/onaio/zebra/blob/master/src/ona/utils/walkthrough.cljc#L55

I agree, I think we should create the `clj-utils` lib for such util functions.